### PR TITLE
Edit lab

### PR DESCRIPTION
**Pull request by inkrypto**
Removed "Bundle exec" to start rails server.

### CODE REVIEW

**Comment by colinross** [comment]
I have to disagree with the edits specifically to the Lab (0614db8fe9e5)

Running gem-based executables without `bundle exec` may work for you, in your environment. If it does than I assume the executable happens to be installed in your system and does not conflict with your bundle. 
Unfortunately, this is unreliable and also a creator of lots of 'wtf' moments. 

Running with `bundle exec` will ensure the right gem versions are used according to the Gemfile.lock and avoid any issues along those lines. This is essentially WHY bundle exists in the first place-- to ensure consistency of gem versions across environments.

tl;dr 
Even if it looks like it works without `bundle exec`, it may not work in the future or on another machine without `bundle exec`.

As for using the `rails s` shortcut, well, I feel keeping the ruby/rails stuff as explicit as possible is better in the context of the bootcamp labs as some of the participants are literally using rails for the first time. 

Just my own two cents.

**Comment by pbnj** [comment]
For best practices, we will keep `bundle exec`.
As for command shortcuts (e.g. `rails s`), we can educate users about this feature, but we should keep the full commands for reference-sake.
